src/windows/CameraProxy.js

```diff
@@ -438,6 +457,61 @@ function takePictureFromCameraWP(successCallback, errorCallback, args) {
             });
     };
 
+    var getAspectRatios = function (capture) {
+        var photoAspectRatios = capture.videoDeviceController.getAvailableMediaStreamProperties(Windows.Media.Capture.MediaStreamType.photo).map(function (element) {
+            return (element.width / element.height).toFixed(1);
+        }).filter(function (element, index, array) { if (index === array.indexOf(element)) return 1; return 0; });
```

Comment:
nit: it should be enough simply return result of comparison from filter function, like `return index === array.indexOf(element)`